### PR TITLE
end user shall use "make llmstudio" to run without wave checking for file changes

### LINKED ISSUE #267: [BUG] Wave reload triggers when uploading the same model twice to Hugging Face
### 🐛 Bug

There is a small issue that I found when trying to upload the same model twice. The wave live update will trigger due to changes in the generated pipeline code.

### To Reproduce

Uplaod same model twice to Hugging Face without deleting the pipeline python file in between.

### PR DESCRIPTION
**Pull request by pascal-pfeiffer**
H2O Wave monitors the app dir for code changes by default since v0.0.9 (https://wave.h2o.ai/blog/release-0.9.0#live-reload).

We create the `h2oai_pipeline.py` file in the output directory when uploading a model to Hugging Face. When doing so a second time, this triggers the reload of wave. 

There is currently no way to disable the check for certain folders, so I decided to disable the reload check completely when not developing. For that, the end user will start the app using `make llmstudio` which starts the wave server without reloading (`--no-reload`).

closes #267

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,8 +80,8 @@ wave:
 	H2O_WAVE_PRIVATE_DIR="/download/@$(WORKDIR)/output/download" \
 	$(PIPENV) run wave run app
 
-.PHONY: wave-no-reload
-wave-no-reload:
+.PHONY: llmstudio
```
Comment (maxjeblick) [inline Makefile:83]: Like the new name!  Should we remove or rename `make wave`, e.g. `make llmstudio-reload` ?

Comment (pascal-pfeiffer) [reply Makefile:83]: feels natural :) 

actually, `make wave` is already in my muscle memory and quite quick to type, so I kind of like it for development. But I would also be fine with `make llmstudio-reload` if you have a strong preference.

Comment (maxjeblick) [reply Makefile:83]: Yeah, I'm also used to it :D Thus, free to leave as is.

**Review by maxjeblick — APPROVED** [review]
lgtm thanks

**Pull request merged**